src/pages/interviews/ditchinghourly.mdx

```diff
@@ -17,3 +17,44 @@ Software engineer, author, and educator Swizec Teller joined me to talk about co
 ></iframe>
 
 [Ditching Hourly – Nov 20, 2021](https://share.transistor.fm/s/3a686c2d)
+
+## Talking Points
+
+- How to apply consulting type thinking to your day job so that you can really rocket ship your career
+- How Swizec sees his Sillicon Valley job
+- Silicon Valley doesn't really hire freelancers
+- Working in SIlicon Valley is very different to the rest of the world
+- The most succesful Silicon Valley people work like consultants
+- Being the employee vs consultant in Silicon Valley
+- Being able to say no to things in the workplace
+- Focussing on results
+- The benefits of using OKRs in the workplace
+- The reason why people change companies often
+
+## Quotable Quotes
+
+"When I made it to Silicon valley, I very quickly realized that the market here doesn't really hire freelancers. At least they try not to hire freelancers for various reasons" - Swizec Teller
+
+"Silicon valley is a little weird in that unless you have Silicon valley experience, they count it as you don't have experience" - Swizec Teller
+
+"Even if you have 20 years experience in different types of industries, once you come to Silicon Valley or to a Silicon Valley-style startup, it essentially resets the zero because the way of working is just that different" - Swizec Teller
```

Comment:
```suggestion
"Even if you have 20 years experience in different types of industries, once you come to Silicon Valley or to a Silicon Valley-style startup, it essentially resets to zero because the way of working is just that different" - Swizec Teller
```